pages/common/help2man.md

```diff
@@ -0,0 +1,35 @@
+# help2man
+
+> Produce simple man pages from an executable's `--help` and `--version` output.
+
+- Generate a man page for an executable:
+
+`help2man {{executable}}`
+
+- Specify the "name" paragraph in the man page:
```

Comment:
What does this do, exactly? I'm not quite sure I understand.

Reply:
It's more of a description. At the top of the man page, it has a `NAME` section. This will something like the following:

```
NAME
       {{executable}} - {{description}}
```

This option basically specifies what goes in the `{{description}}` space.

Reply:
Ahhh ok!